miner/worker.go

```diff
@@ -909,6 +910,13 @@ func (w *worker) commitTransactions(env *environment, txs *types.TransactionsByP
 		if tx == nil {
 			break
 		}
+		// [Kroma: START]
```

Comment:
We've marked the features that is derived from `scroll` with this comment!

```suggestion
		// [Scroll: START]
```